Firmware/Dcodes.cpp

```diff
@@ -922,5 +924,45 @@ void dcode_9125()
 }
 #endif //PAT9125
 
-
 #endif //DEBUG_DCODES
+
+#ifdef XFLASH_DUMP
+#include "xflash_dump.h"
+
+void dcode_20()
+{
+    if(code_seen('E'))
+        xfdump_full_dump_and_reset();
+    else
+    {
+        unsigned long ts = millis();
```

Comment:
Proposal: let’s add an elapsed() member function to Timer.cpp/h. There are so many cases where this would come useful in the firmware.

Reply:
Agreed. I didn't want to add too much fire in a single PR.
Can you make a separate PR for it?